deployment/helm/values.yaml

```diff
@@ -8,11 +8,11 @@ replicaCount: 2
 # -- image parameters
 image:
   # -- The image registry to pull from
-  registry: falco.docker.scarf.sh
+  registry:  docker.io
   # -- The image repository to pull from
   repository: issif/falco-talon
   # -- Override the image tag to pull
-  tag: ""
+  tag: "0.1.0"
```

Comment:
don't change these 2 values please, the `scarf.sh` is there to track the count of pulls (fully anonymous, GDPR compliant), and the tag is managed by a default value set like this:
`{{ default .Chart.AppVersion .Values.image.tag }}`

Reply:
Ahh sorry, feel free to revert it.

Reply:
you need to do it in your branch, I can't merge it with these changes. Sorry

Reply:
ok hope I did it right.